src/index.ts

```diff
@@ -63,11 +63,12 @@ export interface MandeInstance {
    *   // do something
    * })
    * ```
-   * @param url - relative url to send the request to
+   * @param url - optional relative url to send the request to
    * @param options - optional {@link Options}
    */
   get(url: string | number, options: Options<'response'>): Promise<Response>
   get(url: string | number, options: Options<'text'>): Promise<string>
+  get<T = unknown>(options?: Options): Promise<T>
```

Comment:
this probably also needs the other overrides as well and some test types

Reply:
You mean `Options<'response'>` and `Options<'text'>`.

And test types, I don't follow?